tests/relay_integration/lang/javascript/test_plugin.py

```diff
@@ -148,7 +149,7 @@ def test_source_expansion(self, mock_fetch_file):
             },
         }
 
-        mock_fetch_file.return_value.body = "\n".join("hello world")
+        mock_fetch_file.return_value.body = force_bytes("\n".join("hello world"))
```

Comment:
afaict there isn't just a way to join a byte string

```
>>> b"\n".join(b"hello world")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: sequence item 0: expected a bytes-like object, int found
```

Reply:
I mean I found ways but yours is the cleanest. Maybe, just add a comment here?

Reply:
I think no need for a comment, since `force_bytes` is pretty clear what it does